<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>traj_dialog</class>
 <widget class="QDialog" name="traj_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Molecules Controller</string>
  </property>
  <widget class="QPushButton" name="PrevButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>50</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NextButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>50</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>22</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>19</y>
     <width>351</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QDial" name="speedDial">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>250</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>500</number>
   </property>
   <property name="invertedAppearance">
    <bool>true</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="playStopButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>50</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="overlayButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Show all</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>PrevButton</tabstop>
  <tabstop>NextButton</tabstop>
  <tabstop>playStopButton</tabstop>
  <tabstop>speedDial</tabstop>
  <tabstop>verticalSlider</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
